ui/raidboss/data/05-shb/raid/e12s.js

```diff
@@ -80,20 +84,23 @@ const primalOutputStrings = {
   'combined': {
     en: '${safespot1} + ${safespot2}',
     de: '${safespot1} + ${safespot2}',
+    fr: '${safespot1} + ${safespot2}',
     ja: '${safespot1} + ${safespot2}',
     cn: '${safespot1} + ${safespot2}',
     ko: '${safespot1} + ${safespot2}',
   },
   'stock': {
-    en: 'Stock: ${text}',
+    en: 'Stack: ${text}',
```

Comment:
Looking at the translations, I thought it was a mistake. Is it good ?

Reply:
It is good, `Stock` is the skill name in English, (see: https://xivapi.com/Action/22674),
So that I think you should write `Stocker` in fr.

Reply:
it's corrected, Thank you.